pkg/interface/src/logic/subscription/global.ts

```diff
@@ -43,12 +43,25 @@ export default class GlobalSubscription extends BaseSubscription<StoreState> {
     groups: []
   };
   start() {
-    this.subscribe('/all', 'invite-store');
+    this.scry('/all', 'invite-store');
+    this.subscribe('/updates', 'invite-store');
+
+    this.scry('/groups', 'group-store');
     this.subscribe('/groups', 'group-store');
+
     this.subscribe('/primary', 'contact-view');
-    this.subscribe('/all', 'metadata-store');
+
+    this.scry('/associations', 'metadata-store');
+    this.subscribe('/updates', 'metadata-store');
+
+    this.scry('/credentials', 's3-store');
+    this.scry('/configuration', 's3-store');
     this.subscribe('/all', 's3-store');
+
+    this.scry('/tiles', 'launch');
+    this.scry('/first-time' 'launch');
     this.subscribe('/all', 'launch');
```

Comment:
Again, `/all` subscriptions undermine the purpose of scrying for this. They return the same data.

Reply:
The only other subscription available in launch is currently /keys which does not appear to include what you actually want when updates happen. But I don't understand any of these apps so maybe it does.